aspnetcore/fundamentals/host/hosted-services/samples/3.x/BackgroundTasksSample/Services/MonitorLoop.cs

```diff
@@ -24,13 +24,13 @@ public class MonitorLoop
 
         public void StartMonitorLoop()
         {
-            _logger.LogInformation("Monitor Loop is starting.");
+            _logger.LogInformation("MonitorAsync Loop is starting.");
 
             // Run a console user input loop in a background thread
-            Task.Run(() => Monitor());
+            MonitorAsync();
```

Comment:
@davidfowl Does this change look OK to you? I understand it that this means the `MonitorAsync` method will run synchronously up until a key is pressed, but it wouldn't have done so with the previous `Task.Run` wrapper.

Reply:
That's fair, lets revert this change.